Added base64 credentials encoding (#613)
metal3-io/baremetal-operator: docs/api.md

    credentialsVersion: "5562"
```

> And here is the secret `bmo-master-0-bmc-secret` holding the host's BMC credentials, base64 encoded:

Reviewer: ```suggestion
And here is the secret `bmo-master-0-bmc-secret` holding the host's 
BMC credentials, base64 encoded:
```

> `$echo -n 'admin' | base64`
> 
> `YWRtaW4=`
> 
> `$echo -n 'password' | base64`
> 
> `cGFzc3dvcmQ=`

Reviewer: ```suggestion
```console
$echo -n 'admin' | base64

YWRtaW4=

$echo -n 'password' | base64

cGFzc3dvcmQ=
```
```

> Copy the above base64 encoded username and password pair and paste it in the yaml as mentioned below.

Reviewer: ```suggestion
Copy the above base64 encoded username and password pair and 
paste it into the yaml as mentioned below.
```

> And here it is the secret `bmo-master-0-bmc-secret` holding the host's BMC credentials:

Reviewer: ```suggestion
And here is the secret `bmo-master-0-bmc-secret` holding the host's BMC credentials, base64 encoded:
```
